src/cf_cre.erl

```diff
@@ -162,7 +162,8 @@ when is_tuple( App ),
     false ->
 
       % start process
-      _Pid = spawn_link( fun() -> stage( Lam, Fa, DataDir, Mod, R, LibMap, ModState ) end ),
+      Runtime = self(),
+      _Pid = spawn_link( fun() -> stage( Runtime, Lam, Fa, DataDir, Mod, R, LibMap, ModState ) end ),
```

Comment:
I guess here `apply( M, F, A)` doesn't work because `stage/8` is not exported?

I tried but dialyzer complained...